Cargo.toml

```diff
@@ -10,18 +11,16 @@ readme = "README.md"
 repository = "https://github.com/emk/compose_yml"
 
 [dependencies]
-clippy = { version = "0.*", optional = true }
-error-chain = "0.5"
-lazy_static = "1.0"
-log = "0.3"
-regex = "0.2"
-serde = "1.0"
-serde_derive = "1.0"
-serde_json = "1.0"
-# See https://github.com/kbknapp/clap-rs/issues/747 before upgrading to
-# serde_yaml 0.5.x.
-serde_yaml = "0.7"
-yaml-rust = "0.4"
-url = "1.2"
-valico = "1.0"
-void = "1.0"
+clippy = { version = ">=0", optional = true }
+error-chain = ">=0.12.1"
```

Comment:
These open-ended `>=` version dependencies would mean that `compose_yml` would attempt to use **any** future version of a library, including ones with completely-incompatible API changes.

So we need to specify versions as `0.12.1`, and allow Cargo to automatically choose the latest compatible version of the library.

Reply:
I'll fix it